src/components/ui/textarea.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+
+import { cn } from "@/lib/utils"
+
+const Textarea = React.forwardRef<
+  HTMLTextAreaElement,
+  React.ComponentProps<"textarea">
+>(({ className, ...props }, ref) => {
+  return (
+    <textarea
```

Comment:
The Textarea component does not look the same as we have in the design. The Textarea component should be auto-resizable. Manual vertical resizing should be disabled.